index.html

```diff
@@ -2943,6 +2944,229 @@ <h5><code>unsubscribeallevents</code></h5>
+        </p>
+  
+        <figure id="sequence-diagram-webhook">
+          <img src="images/Sequence Diagram - WebHook.png" class="wot-profiles" alt="Webhook example - sequence diagram" />
+          <figcaption>HTTP Webhook Profile - Sequence of operations and notifications</figcaption>
+        </figure>
+  
+  
+        <p>The following example contains a snippet from a TD that illustrates how a Webhook event can be described.</p>
+        <pre class="example">
+        ...
+  
+        {
+            "events": {
+                "fireAlarm": {
+                    "description": "Fire!",
+                    "subscription": {
+                        "type": "object",
+                        "properties": {
+                               ...
+                               listener: uri,
+                               ...
+                            }
+                        }
+                    },
+                   "data": {
+                        "type": "boolean",
+                        "description": "true, if the alert button has been pushed, false, if the button was armed again."
+                    },
+                    "dataResponse": {
+                        "type": "string",
+                        "description": "acknowledged by operator named: "
+                    },
+        
+                    "cancellation": {
+                        "type": "object",
+                         "properties": {
+                               listener: uri
+                            }
+                        }
+                    },
+                    "forms": [
+                        {
+                            "op": "subscribeevent",
+                            "href": "http://192.168.0.124:8080/events/fireAlarm",
+                            "contentType": "application/json",
+                            "htv:methodName": "POST"
+                        },
+                        {
+                            "op": "unsubscribeevent",
+                            "href": "http://192.168.0.124:8080/events/fireAlarm/unsubscribe",
+                            "htv:methodName": "DELETE"
+                        }
+                    ]
+                },
+                "batteryLow": {
+                    ...
+                }
+            }
+        }
+        </pre>
+    
+  
+      </section>
+  
+
+      <section id="http-webhook-profile-protocol-binding-events">
+        <h3>Events</h3>
+        <p>
+          The HTTP Webhook Profile uses Webhooks as a
+          mechanism for Consumers to subscribe to events emitted by a Web Thing.
+        </p>
+        <p class="note">
+          The mechanism is scalable and supports Consumers that receive events from
+          multiple Things. Consumers implement a Webhook listener that is handling the event 
+          streams generated by Things. 
```

Comment:
What does "scaleable" mean here?

The common understanding of the term "stream" within the context of HTTP involves multiple messages being sent over a single HTTP connection, which I don't think applies to Webhooks.

Reply:
I added a clarification to the new draft, please review again.